bin/faucet/src/faucet/store.rs

```diff
@@ -46,7 +47,9 @@ impl FaucetDataStore {
     }
 }
 
+#[maybe_async_trait(?Send)]
 impl DataStore for FaucetDataStore {
+    #[maybe_async]
     fn get_transaction_inputs(
```

Comment:
Why do we need to mark these as `maybe_async`? Should we always have them as `async`?

Reply:
My bad, it can be just `#[async_trait::async_trait(?Send)]` and the method just be async. I will address it in one of my open PRs.